2 - RTB/Documentazione esterna/Piano di Progetto/Piano di Progetto.typ

```diff
@@ -485,3 +485,65 @@ A causa dei rallentamenti incontrati, alcuni ruoli hanno visto un monte ore effe
 Il ruolo dell'amministratore invece, incaricato di gestire il reparto tecnologico implementando gli automatismi necessari, a causa di difficoltà incontrate nell'implementazione (errori, testing, verifica), ha visto un aumento delle ore rispetto a quanto preventivato.
 === Burndown-chart
 [da inserire]
+
+
+
+
+== Quarto Sprint dal 27-11-2023 al 04-12-2023
+
+=== Obiettivi raggiunti
+- effettuare una migliore pianificazione;
+- perfezionamento della stesura degli use case indivduati finora;
+- redazione di una prima versione dell'Analisi dei Requisiti;
+- valutazione di un'eventuale data per un primo meeting con il professore Cardin per ottenere un feedback sull'Analisi dei Requisiti;
+- proseguimento della redazione e aggiornamento del documento di Norme di Progetto;
+- proseguimento periodo di studio di Three.js, in particolare finalizzato a:
+ - creazione di alcuni scaffali nell'ambiente tridimensionale;
+ - implentazione sistema di drag and drop;
+ - parametrizzazione degli elementi presenti nell'applicazione.
+=== Obiettivi mancati
+Tutti gli obiettivi sono stati raggiunti.
+=== Analisi problematiche insorte
+Il team ha notato in fase di retrospettiva una lentezza in fase di review e verifica, che portano al rallentamento del lavoro. Le review
+sono state rallentate anche dalla presenza di numerosi merge conflicts, che il team si è ritrovato a gestire per la prima volta nel progetto.\
+Inoltre si è presa coscienza di una necessità di apportare migliorie al processo di gestione di Jira.
+=== Risoluzioni attuate
+Il team ha preso in considerazione l'utilizzo di Graphite per velocizzare il processo di review e creare pull request brevi.\
+Inoltre il gruppo ha preso l'impegno di utilizzare Jira con più attenzione, per migliorare l'amministrazione del progetto.
```

Comment:
se riesci lo scriverei in maniera leggermente più formale (più attenzione non significa granchè)